app/controllers/hyrax/admin/analytics/collection_reports_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+module Hyrax
+  module Admin
+    module Analytics
+      class CollectionReportsController < AnalyticsController
+        include Hyrax::BreadcrumbsForCollectionAnalytics
+        def index
+          return unless Hyrax.config.analytics == 'true'
+
+          @pageviews = Hyrax::Analytics.daily_events('collection-page-view')
+          @work_page_views = Hyrax::Analytics.daily_events('work-in-collection-view')
+          @downloads = Hyrax::Analytics.daily_events('work-in-collection-download')
+          @all_top_collections = Hyrax::Analytics.top_events('work-in-collection-view', date_range)
+          @top_collections = paginate(@all_top_collections, rows: 10)
+          @top_downloads = Hyrax::Analytics.top_events('work-in-collection-download', date_range)
+          @top_collection_pages = Hyrax::Analytics.top_events('collection-page-view', date_range)
+          respond_to do |format|
+            format.html
+            format.csv { export_data }
+          end
+        end
+
+        def show
+          @document = ::SolrDocument.find(params[:id])
```

Comment:
do we want to populate `@document` even if analytics is turned off?

Reply:
probably not.  removed!

Reply:
on a closer look, i'm wondering if this is a repeated query due to `AnalyticsController#set_document`? it looks like that is running as a `before_action` for `#show`?